Jenkinsfiles/maint-mode.yml

```diff
@@ -0,0 +1,3 @@
+pipeline:
```

Comment:
@escattone I believe, this code should not live in master branch.
I think you should find a way to not merge this to master!
maybe after approving the PR, you can remove the file and make a commit?